src/Serilog/Configuration/LoggerSinkConfiguration.cs

```diff
@@ -149,7 +161,21 @@ internal LoggerSinkConfiguration(LoggerConfiguration loggerConfiguration, Action
             LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum)
         {
             if (logger == null) throw new ArgumentNullException(nameof(logger));
-            return Sink(new SecondaryLoggerSink(logger, attemptDispose: false), restrictedToMinimumLevel);
+            ILogEventSink secondarySink = new SecondaryLoggerSink(logger, attemptDispose: false);
+
+            if (logger is Logger concreteLogger)
+            {
+                if (concreteLogger.HasOverrideMap)
+                {
+                    // Overrides in sub-loggers are not supported and will be ignored
+                    SelfLog.WriteLine("We have detected that you are using .MinimumLevel.Override() from a sub-logger (.WriteTo.Logger(Logger)). " +
+                                      "Overrides in a sub-logger are not supported and are therefore ignored. " +
+                                      "We recommend applying .MinimulLevel.Override on the root logger or " +
+                                      "using .Filter in your sub-loggers. " +
+                                      "Note that support for sub-loggers' minimum level overrides may be removed in the next major version.");
```

Comment:
Dropping the comment and using the shorter message would reduce the sense of duplication here.